packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission-for-mt5.jsx

```diff
@@ -16,21 +16,23 @@ const POISubmissionForMT5 = ({
     refreshNotifications,
     citizen_data,
     has_idv_error,
-    jurisdiction_selected_shortcode,
+    residence_list,
+    account_settings,
+    landing_company_shortcode,
 }) => {
     const [submission_status, setSubmissionStatus] = React.useState(); // submitting
     const [submission_service, setSubmissionService] = React.useState();
     React.useEffect(() => {
         if (citizen_data) {
             const { submissions_left: idv_submissions_left } = idv;
             const { submissions_left: onfido_submissions_left } = onfido;
-            const is_idv_supported = citizen_data.identity.services.idv.is_country_supported;
-            const is_onfido_supported = citizen_data.identity.services.onfido.is_country_supported;
+            const is_idv_supported = isVerificationServiceSupported(residence_list, account_settings, 'idv');
+            const is_onfido_supported = isVerificationServiceSupported(residence_list, account_settings, 'onfido');
             if (
                 is_idv_supported &&
                 Number(idv_submissions_left) > 0 &&
                 !is_idv_disallowed &&
-                jurisdiction_selected_shortcode !== 'vanuatu'
+                landing_company_shortcode !== 'maltainvest'
```

Comment:
why do we have landing_company_shortcode here?

Reply:
As we can't rely on is_idv_disallowed, we need to trigger onfido if the client supports both for DIEL accounts like South Africa, so adding the check here.